<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Managerview</class>
 <widget class="QDialog" name="Managerview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理我的程序</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="launchButton">
       <property name="text">
        <string>启动程序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>关闭程序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gotoButton">
       <property name="text">
        <string>查看程序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
